ToDoList/test/ToDoList.API.Services.Tests/UrlLocationServiceTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Net.Http;
+using System.Web.Http;
+using System.Web.Http.Routing;
+using NSubstitute;
+using NUnit.Framework;
+
+using ToDoList.Contracts.Services;
+
+namespace ToDoList.API.Services.Tests
+{
+    [TestFixture]
+    public class UrlLocationServiceTests
+    {
+        private IRoutesService _routesService;
+        private UrlLocationService _locationService;
+
+        private const string WhatsUp = "whatsUp";
+
+        [SetUp]
+        public void SetUp()
+        {
+            var configuration = new HttpConfiguration();
+            configuration.Routes.MapHttpRoute("WhatsUp", $"{WhatsUp}/{{id}}", new { id = RouteParameter.Optional });
```

Comment:
Please, add something to follow the `id` to prevent code that only suffixes what-ever path with `id`…